bindings/Objective-C/MEGAChatSdk.mm

```diff
@@ -921,6 +921,26 @@ - (void)removeUnsentMessageForChat:(uint64_t)chatId rowId:(uint64_t)rowId {
     self.megaChatApi->removeUnsentMessage(chatId, rowId);
 }
 
+- (void)addReactionForChat:(uint64_t)chatId messageId:(uint64_t)messageId reaction:(NSString *)reaction {
+    self.megaChatApi->addReaction(chatId, messageId, reaction ? [reaction UTF8String] : NULL);
```

Comment:
Is it possible to replace 
` reaction ? [reaction UTF8String] : NULL`
With
`reaction.UTF8String`
as performing any operation on nil would return nil. I think we don't have to check for the condition. Let me know your thoughts.

If it possible I think we can replace it everywhere on your PR.

Reply:
Yes, it is possible if reaction is nil it will pass NULL as parameter, so we can replace `reaction ? [reaction UTF8String] : NULL` with `reaction.UTF8String`